test/sys/test_epoll.rs

```diff
@@ -1,26 +1,23 @@
-#![allow(deprecated)]
-
-use nix::errno::Errno;
-use nix::sys::epoll::{epoll_create1, epoll_ctl};
-use nix::sys::epoll::{EpollCreateFlags, EpollEvent, EpollFlags, EpollOp};
+use nix::{
+    errno::Errno,
+    sys::epoll::{Epoll, EpollCreateFlags, EpollEvent, EpollFlags},
+};
+use std::os::fd::BorrowedFd;
 
 #[test]
 pub fn test_epoll_errno() {
-    let efd = epoll_create1(EpollCreateFlags::empty()).unwrap();
-    let result = epoll_ctl(efd, EpollOp::EpollCtlDel, 1, None);
-    result.expect_err("assertion failed");
+    let epoll = Epoll::new(EpollCreateFlags::empty()).unwrap();
+    let fd_1 = unsafe { BorrowedFd::borrow_raw(1) };
+    let result = epoll.delete(fd_1);
     assert_eq!(result.unwrap_err(), Errno::ENOENT);
-
-    let result = epoll_ctl(efd, EpollOp::EpollCtlAdd, 1, None);
-    result.expect_err("assertion failed");
-    assert_eq!(result.unwrap_err(), Errno::EINVAL);
 }
 
 #[test]
-pub fn test_epoll_ctl() {
-    let efd = epoll_create1(EpollCreateFlags::empty()).unwrap();
-    let mut event =
-        EpollEvent::new(EpollFlags::EPOLLIN | EpollFlags::EPOLLERR, 1);
-    epoll_ctl(efd, EpollOp::EpollCtlAdd, 1, &mut event).unwrap();
-    epoll_ctl(efd, EpollOp::EpollCtlDel, 1, None).unwrap();
+pub fn test_epoll_add_delete() {
+    let epoll = Epoll::new(EpollCreateFlags::empty()).unwrap();
+    let event = EpollEvent::new(EpollFlags::EPOLLIN | EpollFlags::EPOLLERR, 1);
+    let fd_1 = unsafe { BorrowedFd::borrow_raw(1) };
```

Comment:
Just to get rid of the unsafe, I suggest:
```suggestion
    let stdin = std::io::stdin();
    let fd_1 = stdin.as_fd();
```

Reply:
will do